manage/knowledge-base.rst

```diff
@@ -197,3 +197,33 @@ Delete
 Use this panel to delete the knowledge base.
 If you wish to unpublish it without deleting all of its content,
 simply disable it via the toggle button at the top of the window instead.
+
+Permissions
+-----------
+
+The general access to the knowledge base is managed via
+``knowledge_base.reader`` and ``knowledge_base.editor`` permissions. You can
+find more information about roles and permissions
+:doc:`here <roles/index>`.
+
+If you want to grant more granular permissions than the global read/edit
+permissions (e.g. edit permissions only for specific parts of the knowledge
+base), you can do so by following the steps below:
+
+- First create the base structure of the knowledge base (if not already done)
+- Create or choose one or more roles which should have granular permissions
+- Grant the ``knowlege_base.reader`` permission to the desired role(s)
+- Edit the permissions of each (sub-)category of the knowledge base
+- Assign the role to one or more users who should be able to edit the knowledge
+  base (partially)
+
+.. note::
+
+   - The permissions of a parent category are inherited to sub-categories.
+   - The permission ``knowledge_base.reader`` can be widened and restricted.
+   - The permission ``knowledge_base.editor`` **can't** be adjusted.
```

Comment:
I'm not sure about those lines. I wonder if it's clear enough that we're talking about permission up/down grades in sub-categories?

Maybe something like this would be more clear?  Or a different phrasing to say „down the tree“ :)

```
- The permission ``knowledge_base.reader`` can be widened or restricted in sub-categories
- The permission ``knowledge_base.editor`` **can't** be adjusted in sub-categories.
```